pkg/power/host.go

```diff
@@ -41,9 +41,8 @@ func initHost(nodeName string) (Host, error) {
 	host := &hostImpl{
 		name:           nodeName,
 		exclusivePools: PoolList{},
-		featureStates:  &featureList,
 	}
-
+	host.featureStates = &featureList
```

Comment:
I can't remember exactly why I did this but I think it was because I was getting some weird behavior where the host was using a snapshot of the pointers' value instead of the pointer itself. Think this was just for the purposes of debugging to see the field go from unassigned to assigned but probably best to leave as is right now just in case and revert it for next time around